changelog.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+
+## 0.15.0
+
+This release is brings download progress bars for file downloads
```

Comment:
"This release vastly expands printer options by using IPP-over-USB to support any AirPrint-capable printer, in addition to switching to the modern GTK print dialog. Also, file downloads now show a progress bar to give users visual feedback."

Or something like that :)